passenger_exporter_test.go

```diff
@@ -224,6 +257,7 @@ func TestInsertingNewProcesses(t *testing.T) {
 			Process{PID: "newPID"},
 			Process{PID: "newPID2"},
 		},
+		6,
```

Comment:
How have you come up with `6` here?

Reply:
See https://github.com/Intellection/passenger-exporter/pull/14#discussion_r394157681.